dvc/command/install.py

```diff
@@ -27,4 +27,11 @@ def add_parser(subparsers, parent_parser):
         help=INSTALL_HELP,
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
+    install_parser.add_argument(
+        "--use-pre-commit-tool",
```

Comment:
It'd be good to have `--use-pre-commit` or just `--pre-commit`. Tool is kind of redundant here.

Reply:
@skshetry Im worried about it being confused with `pre-commit` hooks.